### DISCUSSION TITLE
netboot will be added to quickemu

### DISCUSSION THREAD

**Discussion by zen0bit** [antonym/netboot.xyz#1074] [Show and tell]
It's in pull requests
[quickemu github](https://github.com/quickemu-project/quickemu/pull/346)
You can check if I didn't make mistake...

Just let you know...

And thanks for super project...

**Comment by antonym** [comment]
Interested in seeing support... could you refer to it as `netboot.xyz` though? Thanks!

**Reply by zen0bit** [reply]
Like you want sir...
